geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -366,7 +373,33 @@ app.DrawController.prototype.onDrawEnd_ = function(event) {
         ol.geom.Polygon.fromCircle(featureGeom)
     );
   }
-  feature.set('name', 'element ' + (++this.featureSeq_));
+
+  /**
+   * @type {string}
+   */
+  var name;
+  switch (feature.getGeometry().getType()) {
+    case 'Point':
+      if (this.drawLabel.getActive()) {
+        name = this.translate_.getString('Label');
+      } else {
+        name = this.translate_.getString('Point');
+      }
+      break;
+    case 'LineString':
+      name = this.translate_.getString('LineString');
+      break;
+    case 'Polygon':
+      name = this.translate_.getString('Polygon');
+      break;
+  }
+  feature.set('name', name + ' ' + (++this.featureSeq_));
+  this.drawnFeatures_.features.forEach(function(drawnFeature) {
```

Comment:
The `features` should be private in `drawnfeaturesservice` and we should use `getCollection` instead.